bin/morgue.js

```diff
@@ -4719,7 +4808,44 @@ function attributeSet(argv, config, opts) {
     },
   });
 
-  bpgPost(state.bpg, request, bpgCbFn('Attribute', 'update'));
+  bpgPost(state.bpg, request, bpgCbFn('View', 'update'));
+}
+
+function viewCreate(argv, config, opts) {
+  const state = opts.state;
+
+  if (!argv.queries) return viewUsageFn("Must specify queries.");
+  if (!argv.payload) return viewUsageFn("Must specify payload.");
+
+  // json parse or keep input as json
+  const queries = typeof argv.queries === 'string' ? JSON.parse(argv.queries) : argv.queries
+  const payload = typeof argv.payload === 'string' ? JSON.parse(argv.payload) : argv.payload
+  // update name
+  payload.name = opts.params.attrname
+
+  const request = bpgSingleRequest({
+    action: "create",
+    type: "configuration/query",
+    object: {
+      name: opts.params.attrname,
+      project: state.project.fields.pid,
+      owner: config.config.uid,
```

Comment:
We probably want to validate that these fields exist before accessing to prevent accidental runtime errors